Packs/MicrosoftExchangeOnline/Integrations/EWSO365/README.md

```diff
@@ -1,135 +1,142 @@
-You can't manage the **Office 365 Exchange Online** app permissions via the Azure portal.
+For more information on this permission, visit [the Microsoft documentation](https://docs.microsoft.com/en-us/exchange/client-developer/exchange-web-services/how-to-authenticate-an-ews-application-by-using-oauth#configure-for-app-only-authentication).
 
 To limit the application's permissions to only specific mailboxes, follow the [Microsoft documentation](https://docs.microsoft.com/en-us/graph/auth-limit-mailbox-access). Note that it may take about an hour for permissions changes to take effect.
 
-## Fetched Incidents Data
+## Configure Integration on Cortex
+
+| **Parameter** | **Description** |**Required**|
+| --- | --- | --- |
+| ID / Application ID | ID can be received after following the System Integration Setup (Device side steps). | False |
+| Token / Tenant ID | Token can be received after following the System Integration Setup (Device side steps). | False |
+| Key / Application Secret | Key can be received after following the System Integration Setup (Device side steps). | False |
+| Azure Cloud | Azure Cloud environment. Options are: _Worldwide_ (The publicly accessible Azure Cloud), _US GCC_ (Azure cloud for the USA Government Cloud Community), _US GCC-High_ (Azure cloud for the USA Government Cloud Community High), _DoD_ (Azure cloud for the USA Department of Defense), _Germany_ (Azure cloud for the German Government) | False|
+| Email Address | Mailbox to run commands on and to fetch incidents from. To use this functionality, your account must have impersonation rights or delegation for the account specified. For more information, see https://xsoar.pan.dev/docs/reference/integrations/ewso365/#additional-information | True |
+| UPN Address | When provided, the target mailbox if it's different from the Email Address. Otherwise, the Email Address is used. | False |
+| Name of the folder from which to fetch incidents | Supports Exchange Folder ID and sub-folders, e.g., Inbox/Phishing. | True |
+| Access Type | Run the commands using `Delegate` or `Impersonation` access types. | False |
+| Public Folder |  Whether the folder to be fetched from is public. Public folders can store and organize emails on specific topics or projects. Public folders are usually listed under the "Public Folders" section in the navigation pane in the product itself. | False |
+| Fetch incidents |  | False |
+| Incident type |  | False |
+| First fetch timestamp (&lt;number&gt; &lt;time unit&gt;, e.g., 12 hours, 7 days) |  | False |
+| Maximum number of incidents per fetch (up to 200). Performance might be affected by a value higher than 50. |  | False |
+| Mark fetched emails as read | Mark fetched emails as read. | False |
```

Comment:
The description does not add anything since it is just a repeat of the parameter. I recommend just remove the description.